CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## 1.17.3-dev
+* Enhancement - Wire context into methods that invoke Docker API call to fix flaky tests [#1329](https://github.com/aws/amazon-ecs-agent/pull/1321)
```

Comment:
I don't think we normally update Changelog for internal text fixes like this. Also, 1.17.3 just got released so it shouldn't be under 1.17.3-dev.

Reply:
OK, I will remove it.